config/locales/en.yml

```diff
@@ -62,6 +62,7 @@ en:
     allowed_access_to_user: '%{admin_name} allowed access'
     denied_access_to_user: '%{admin_name} denied access'
     dashboard_title: "Simple Server"
+    phi_access_alert: "You are about to view confidental patient data. Are you sure you want to proceed?"
```

Comment:
I think `about to view confidential data` should be changed to `about to download confidential data`, as they are already viewing the data when they click the button. This will make it clearer that they are going to get a copy of confidential data on their machine.